### ISSUE TITLE
Window shadow not scaling when monitor scale not 100%

### ISSUE THREAD

**Issue by jnko** [nclarius/Plasma-window-decorations#2]
(well probably this may be a KDE bug)

My laptop has a 4K display, so I set my monitor scale to 200%. In this setup the a windows shadow is not scaled to 200% too.
See screenshot

![2022-07-13_13-50](https://user-images.githubusercontent.com/2241599/178728200-e084e5ea-ed63-49aa-bea4-fb15892a841b.png)

**Comment by nclarius** [comment]
Yes, this is an issue of KDE,  not something that can be fixed in the theme. I suggest you file a bug report at bugs.kde.org, product KWin, component Aurorae.
